Correct lint errors introduced by #110

PR #110 introduced 6 ESLint errors (see below). In an effort to avoid the use of `any` for all code introduced after #61 was merged, these should be corrected to have proper types.

```
   30:7   error  Unsafe assignment of an any value                  @typescript-eslint/no-unsafe-assignment
   31:7   error  Unsafe assignment of an any value                  @typescript-eslint/no-unsafe-assignment
   32:7   error  Unsafe assignment of an any value                  @typescript-eslint/no-unsafe-assignment
   34:7   error  Unsafe assignment of an any value                  @typescript-eslint/no-unsafe-assignment
  507:36  error  Invalid type "any" of template literal expression  @typescript-eslint/restrict-template-expressions
  507:44  error  Invalid type "any" of template literal expression  @typescript-eslint/restrict-template-expressions
```

--- a/src/ApiHandler.ts
+++ b/src/ApiHandler.ts
@@ -68,13 +68,13 @@ export default class ApiHandler {
 		]);
 
 		const { parentHash, number, stateRoot, extrinsicsRoot } = block.header;
-		const parentParentHash = await async function() {
+		const parentParentHash = await (async function () {
 			if (block.header.number.toNumber() > 1) {
 				return (await api.rpc.chain.getHeader(parentHash)).parentHash;
 			} else {
 				return parentHash;
 			}
-		}();
+		})();
 
 		const onInitialize = { events: [] as SanitizedEvent[] };
 		const onFinalize = { events: [] as SanitizedEvent[] };

--- a/src/config.json
+++ b/src/config.json
@@ -1,3 +1,3 @@
 {
-  "CUSTOM_TYPES": {},
+  "CUSTOM_TYPES": {}
 }

--- a/src/main.ts
+++ b/src/main.ts
@@ -26,17 +26,23 @@ import ApiHandler from './ApiHandler';
 import * as config from './config.json';
 import { parseBlockNumber, sanitizeNumbers } from './utils';
 
-const HOST = process.env.BIND_HOST || config['BIND_HOST'] || '127.0.0.1';
-const PORT = Number(process.env.BIND_PORT) || config['BIND_PORT'] || 8080;
-const WS_URL = process.env.NODE_WS_URL || config['NODE_WS_URL'] || 'ws://127.0.0.1:9944';
-const LOG_MODE = process.env.LOG_MODE || config['LOG_MODE'] || 'errors';
+const HOST =
+	process.env.BIND_HOST || (config['BIND_HOST'] as string) || '127.0.0.1';
+const PORT =
+	Number(process.env.BIND_PORT) || Number(config['BIND_PORT']) || 8080;
+const WS_URL =
+	process.env.NODE_WS_URL ||
+	(config['NODE_WS_URL'] as string) ||
+	'ws://127.0.0.1:9944';
+const LOG_MODE =
+	process.env.LOG_MODE || (config['LOG_MODE'] as string) || 'errors';
 
 type Params = { [key: string]: string };
 
 async function main() {
 	const api = await ApiPromise.create({
 		provider: new WsProvider(WS_URL),
-		types: config['CUSTOM_TYPES']
+		types: config['CUSTOM_TYPES'],
 	});
 	const handler = new ApiHandler(api);
 	const app = express();

